config-windows.md

```diff
@@ -3,6 +3,22 @@
 This document describes the schema for the [Windows-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
 The Windows container specification uses APIs provided by the Windows Host Compute Service (HCS) to fulfill the spec.
 
+## <a name="configWindowsLayerFolders" />LayerFolders
+
+**`layerFolders`** (array of strings, REQUIRED) specifies a list of layer folders the container image relies on. The list is ordered from topmost layer to base layer.
```

Comment:
This is the first instance (as far as I know) of a REQUIRED property that is a direct child of a platform property (`windows`, in this case).  Because REQUIRED is only “[if the parent is set][1]”, I think you'll want to adjust [this section][2] to say that `windows` MUST be set when `platform.os` is `windows`.

Discussion in #830 may end up with [`platform` being removed entirely][3].  If that lands first, it will change the phrasing for the `windows` requirement, but requiring the `windows` object for configs aimed at the [`windows` platform][4] will still be something you want.

[1]: https://github.com/wking/opencontainer-runtime-spec/blob/05da00bc76e1746276e977ab45957ab39c31c73a/spec.md#property-specification
[2]: https://github.com/opencontainers/runtime-spec/blob/49b0a1fa5f2b6f05195305ebe62d59444ffd73e8/config.md#platform-specific-configuration
[3]: https://github.com/opencontainers/runtime-spec/pull/830#issuecomment-303463569
[4]: https://github.com/opencontainers/runtime-spec/blob/49b0a1fa5f2b6f05195305ebe62d59444ffd73e8/spec.md#platforms